source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -55,103 +96,127 @@ content: |
-           BsonString awsAccessKeyId = new BsonString("<IAM User Access Key ID>");
-           BsonString awsSecretAccessKey = new BsonString("<IAM User Secret Access Key>");
+           // TODO: check correctness
            Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>();
            Map<String, Object> providerDetails = new HashMap<String, Object>();
 
-           providerDetails.put("accessKeyId", awsAccessKeyId);
-           providerDetails.put("secretAccessKey", awsSecretAccessKey);
-           providerDetails.put("region", masterKeyRegion);
+           providerDetails.put("email", new BsonString("<GCP service account email>"));
+           providerDetails.put("privateKey", new BsonString("<GCP service account private key>"));
+           providerDetails.put("endpoint", new BsonString("<GCP authentication endpoint>"));
+
+           kmsProviders.put("gcp", providerDetails);
 
-           kmsProviders.put("aws", providerDetails);
      .. tab::
         :tabid: nodejs
 
         .. code-block:: javascript
 
+           // TODO: check correctness
            kmsProviders = {
-             aws: {
-               accessKeyId: '<IAM User Access Key ID>',
-               secretAccessKey: '<IAM User Secret Access Key>',
+             gcp: {
+               email: '<GCP service account email>',
+               privateKey: '<GCP service account private key>',
+               endpoint: '<GCP authentication endpoint>',
              }
            }
+
      .. tab::
         :tabid: python
 
         .. code-block:: python
 
-           kms_providers = {
-               "aws": {
-                   "accessKeyId": "<IAM User Access Key ID>",
-                   "secretAccessKey": "<IAM User Secret Access Key>"
+           kms_provider = {
+               "gcp": {
+                   "email": "<GCP service account email>",
+                   "privateKey": "<GCP service account private key>",
+                   "endpoint": "<GCP authentication endpoint>",
                }
            }
 ---
 title: Create a New Data Encryption Key
-ref: create-a-new-data-key
+ref: create-a-new-data-key-gcp
 content: |
-  The following diagram shows how the **customer master key** is created and
-  stored when using a KMS provider:
+  In this step, we generate a new **data encryption key** using the
+  **master key** in the remote KMS. The following diagram shows the
+  requests you make from the client application to create and store a
+  new data encryption key:
 
   .. image:: /figures/CSFLE_Data_Key_KMS.png
      :alt: Diagram that describes creating a data encryption key when using a KMS provider
 
-  You must generate a new **data encryption key** using the **master key**
-  in the remote KMS. The original data encryption key was encrypted by
-  your locally-managed master key.
+  Provide your client with the following information to access the master key:
```

Comment:
Suggestion: I wonder if there should be substeps here as well, since you need to do these in order. Also, we should set the condition/context of the sentence first, so I'd flip "to access the master key" to the front. Something like:

a. To access the master key, provide the following information to your client:
    table
b. Update and run the following code to generate a new data encryption key:
    code

Reply:
Thanks, this makes sense. I'm going to move this suggestion to a new issue since it applies to content in multiple places.

Reply:
https://jira.mongodb.org/browse/DOCSP-13089